token/common.tact

```diff
@@ -93,3 +93,33 @@ struct BurnNotification {
 
     @derive impl Serialize {}
 }
+
+contract interface JettonOwner {
+    msg receive_notification(prefix: 0x7362d09c, notification: TransferNotification);
+}
+
+contract interface ResponseReceiver {
+    msg receive_response(prefix: 0xd53276db, response: Excesses);
+}
+
+contract interface Master {
+    msg receive_burn_notification(prefix: 0xd53276db, notification: BurnNotification);
+}
+
+contract interface Jetton {
```

Comment:
```suggestion
interface JettonBalance {
```

I think the current name "jetton wallet" is not terrible (it's just an internal technical jargon), but far from ideal. It is not "jetton" either because jetton is a concept of an asset that is implemented with a number of contracts and interfaces, neither of which deserves a dedicated name "jetton".

What this contract does is holds user's jetton balance. Yes, like a wallet, but the wallet is the ultimate decision maker, while JettonBalance is not - it is fully controlled by another contract (the actual wallet).

Reply:
Well, you may be right, but naming issues are not related to the current PR. I think we can do it in the separate PR.

Reply:
ok, anyhow let's keep simply `interface`, not `contract interface`. For types we could use `traits` - very possible we'd need them mostly for stdlib and most users won't be implementing their types with traits. They'd be implementing contracts with interfaces primarily.

Reply:
Fixed in [1f63906](https://github.com/tact-lang/tact-bikeshed/pull/5/commits/1f63906469f35d23260a3ba0ab4480f450df9520)